src/figuresClasses.ts

```diff
@@ -1,11 +1,99 @@
-export interface Figure {}
+export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  color: 'red' | 'green' | 'blue';
+
+  shape: 'triangle';
```

Comment:
The `shape` property is defined as a literal type 'triangle', but according to the `Figure` interface, it should be of type `string`. Consider changing it to `string` to match the interface.